CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+Contributing and Development
+============================
+This is a quick guide on how to develop and contribute to this project
+
+## Dependencies
+Make sure you can run these commands and install them if not present.
+* python3
+* pip3 (packaged as python3-pip)
+
+
+## Get a Discord bot token
+* Go to https://discordapp.com/developers/applications (log in if needed)
+* Create an application (name doesn't matter)
+* Click "Bot" it the left sidebar
+* Create a bot
+  * The bot's name will be what users see in servers
+  * Changing the bot's name changes the BotToken
+* Make note of the token on this page (later refered to as BotToken)
+
+
+## Join the test server
+[Click here to join the server](https://discord.gg/4xwKBs)
```

Comment:
Here we wil need to mess with channel perms, we cant have all the bots be admins we have to segregate them to their own channels